AraCorrelator/RayTraceCorrelator.cxx

```diff
@@ -275,6 +275,7 @@ std::vector<TGraph> RayTraceCorrelator::GetCorrFunctions(
     // first, calculate all of the correlation functions
     // for performance reasons, it's actually better to store
     // the correlation functions as a vector
+    // std::vector<std::unique_ptr<ROOT::Math::Interpolator>> corrFunctions;
```

Comment:
Are we leaving this in as a reference in case we ever switch to storing interpolators?

Reply:
No, that was a mistake! Will remove.